common/headers.go

```diff
@@ -0,0 +1,53 @@
+package common
+
+import (
+	"fmt"
+	"runtime"
+)
+
+const (
+	HEADER_NAME_USER_AGENT = "User-Agent"
+
+	SDK_NAME = "vpc-beta-go-sdk"
+)
+
+// GetSdkHeaders - returns the set of SDK-specific headers to be included in an outgoing request.
```

Comment:
can these comments be cleaned up

Reply:
this is the explanation of GetSdkHeaders which has parameters and return parameters. This is present in vpc-go-sdk as well. Don't we need this ?